plugins/spreadsheet-view/src/SpreadsheetView/components/ImportWizard.js

```diff
@@ -52,6 +51,7 @@ const NumberEditor = observer(
 )
 
 const ErrorDisplay = observer(({ error }) => {
+  console.log({ error })
```

Comment:
console log may have been left over here

Reply:
+1 that entire component was replaced instead with usage of @jbrowse/core/ui/ErrorMessage. also added some typescripting to that file